CMakeLists.txt

```diff
@@ -156,6 +159,12 @@ IF (ENABLE_ARCHREPO OR ENABLE_DEBIAN)
 SET (ENABLE_LZMA_COMPRESSION ON)
 ENDIF (ENABLE_ARCHREPO OR ENABLE_DEBIAN)
 
+IF (USE_SYSTEM_ZCHUNK)
+SET (ENABLE_ZCHUNK_COMPRESSION ON)
```

Comment:
This check should be inverted. It should be `USE_SYSTEM_ZCHUNK` underneath `ENABLE_ZCHUNK_COMPRESSION`.

Reply:
I think it's right the way it is.  It's saying that if USE_SYSTEM_ZCHUNK is enabled, we should automatically enable ENABLE_ZCHUNK_COMPRESSION.  However, if you're using the embedded zchunk decompressor, ENABLE_ZCHUNK_COMPRESSION should be enabled, and USE_SYSTEM_ZCHUNK shouldn't be.